### PR TITLE
*: cut v1.9.8

### PR DESCRIPTION
**Pull request by tariq1890**

### CODE REVIEW

File: tests/e2e.sh [hunk 1]
```diff
@@ -25,7 +25,7 @@ esac
 
 KUBERNETES_VERSION=v1.18.6
 KUBE_STATE_METRICS_LOG_DIR=./log
-KUBE_STATE_METRICS_IMAGE_NAME="quay.io/coreos/kube-state-metrics-${ARCH}"
+KUBE_STATE_METRICS_IMAGE_NAME="k8s.gcr.io/kube-state-metrics/kube-state-metrics-${ARCH}"
```
Comment (mrueg) [inline tests/e2e.sh:28]: small nitpick: k8s.gcr.io supports multi-arch with 1.9.8 so we could omit the -${ARCH} here

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -194,4 +194,4 @@ jobs:
 
     - name: End-to-end tests
       run: |
-        REGISTRY="quay.io/coreos" make e2e
+        REGISTRY="k8s.gcr.io/kube-state-metrics" make e2e
```
Comment (mrueg) [inline .github/workflows/ci.yml:197]: Does this one still have an effect at all or can we simply remove it?

Comment (mrueg) [reply .github/workflows/ci.yml:197]:
```suggestion
        make e2e
```

File: tests/e2e.sh (hunk 1 above)
Comment (mrueg) [inline tests/e2e.sh:28]:
```suggestion
KUBE_STATE_METRICS_IMAGE_NAME="k8s.gcr.io/kube-state-metrics/kube-state-metrics"
```

**Review by mrueg — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (tariq1890) [reply .github/workflows/ci.yml:197]: Yeah, I don't think it has an effect

File: tests/e2e.sh (hunk 1 above)
Comment (tariq1890) [reply tests/e2e.sh:28]: I am thinking of going back to quay as the default image

**Review by mrueg — APPROVED** [review]

**Comment by mrueg** [comment]
lgtm! Thanks @tariq1890

**Pull request merged**